src/moparser.js

```diff
@@ -131,43 +129,40 @@ Parser.prototype._handleCharset = function (headers) {
     this._charset = this._table.charset = formatCharset(match[1], this._charset);
   }
 
-  headers = encoding.convert(headers, 'utf-8', this._charset)
-    .toString('utf8');
+  headers = encoding.convert(headers, 'utf-8', this._charset);
 
-  this._table.headers = parseHeader(headers);
+  this._table.headers = parseHeader(headers.toString('utf8'));
 };
 
 /**
  * Adds a translation to the translation object
  *
- * @param {String} msgid Original string
- * @params {String} msgstr Translation for the original string
+ * @param {String} msgidRaw Original string
+ * @param {String} msgstrRaw Translation for the original string
```

Comment:
```suggestion
 * @param {string} msgidRaw Original string
 * @param {string} msgstrRaw Translation for the original string
```

All of the build in types should be lowercase, I'll just note it here, but it would be nice to clean it up throughout.